README.md

```diff
@@ -1,3 +1,9 @@
+
+<img src="https://github.com/neuropoly/spinalcordtoolbox/tree/nipin/issue836_axial_rotation/regi1.png" width="300">
```

Comment:
are these binary files uploaded on purpose? we usually try to avoid adding binaries to the repository (unless highly justified)

Reply:
Those png files were on my  "hog" branch, to illustrate my method, don't know why they ended up there, will correct that